vllm/entrypoints/openai/protocol.py

```diff
@@ -172,20 +172,20 @@ class ChatCompletionRequest(OpenAIBaseModel):
     echo: Optional[bool] = Field(
         default=False,
         description=(
-            "If true, the new message will be prepended with the last message "
+            "If True, the new message will be prepended with the last message "
```

Comment:
These classes define the HTTP API JSON scheme so I think `true` is actually more appropriate?